client/packages/programs/src/api/hooks/useProgramList.ts

```diff
@@ -7,7 +7,11 @@ import { ProgramFragment } from '../operations.generated';
 import { useProgramsGraphQL } from '../useProgramsGraphQL';
 import { LIST, PROGRAM } from './keys';
 
-export const useProgramList = (includeImmunisation?: boolean) => {
+export const useProgramList = ({
+  isImmunisation,
+}: {
+  isImmunisation?: boolean;
```

Comment:
Old way only supported "only non-immunisation programs" or "all programs", needed the third option of "only immunisation programs". Felt the boolean parameter was not clear what it meant, so changed to options object